Remotion/Web/Core/Res/Themes/NovaViso/HTML/DropDownMenu.css

```diff
@@ -24,6 +24,36 @@ span.DropDownMenuSelect
   cursor: pointer;
 }
 
+span.DropDownMenuContainer.primary span.DropDownMenuSelect:not(.disabled)
+{
+  --color-standard-text: var(--color-primary-text);
+  background-color: var(--color-primary);
+  border: none;
+}
+
```

Comment:
I'm missing the update to the a.DropDownMenuLabel in primary. we should make this either white or the light-blue variant and then, during hover, the border turns primary-color. quick demo or different screenhots?

Reply:
I don't think I know what the two solutions are that you want to compare.

I have added the color styling and also introduced a border on hover. Let me know if there is something else that I should test

Reply:
That's the one:
![image](https://user-images.githubusercontent.com/6972548/133984525-1a152f5c-d297-4200-bde1-51b2e801252f.png)
![image](https://user-images.githubusercontent.com/6972548/133984627-10713c34-4366-40e1-8f50-2ea1f1579a05.png)